user_interface.py

```diff
@@ -51,23 +57,42 @@ def write_products(file):
         product = catalog.BreadGoods(name, price)
         catalog.products.append(product)
 
-print("Do you wish to write the list into a .txt file")
+
+print("Do you wish to write the list into a database?")
 answer = input("type Y or N ").lower()
 if answer == "y":
-    print("Do you wish to use manual directory input? If not then script's directory will be used")
-    answer = input("type Y or N ").lower()
-    if answer == "y":
-        file_directory = input("Please redeem the directory of the file you are going to use (example D:\list\goods) ")
-        file = f'{file_directory}.txt'
-        write_products(file)
-    elif answer == "n":
-        file_directory = os.getcwd()
-        file_name = input("Please redeem your file's name ")
-        file = f'{file_directory}\{file_name}.txt'
-        write_products(file)
-    else:
-        print("You have to print either Y or N")
+    try:
+        # connect to exist database
+        connection = psycopg2.connect(
```

Comment:
Move it to separate module.
security issues can be ralised with some default values and commant abput needful changes.